vowpalwabbit/core/src/parse_args.cc

```diff
@@ -682,7 +682,8 @@ void parse_feature_tweaks(options_i& options, VW::workspace& all, bool interacti
                .help("Don't remove interactions with duplicate combinations of namespaces. For ex. this is a "
                      "duplicate: '-q ab -q ba' and a lot more in '-q ::'."))
       .add(make_option("quadratic", quadratics).short_name("q").keep().help("Create and use quadratic features"))
-      .add(make_option("cubic", cubics).keep().help("Create and use cubic features"));
+      .add(make_option("cubic", cubics).keep().help("Create and use cubic features"))
+      .add(make_option("indexing", all.indexing).one_of({0, 1}).keep().help("Choose between 0 or 1-indexing"));
```

Comment:
verify that the logic of one_of doesn't clash with the default value of 2 of indexing

Reply:
Yea it works fine if the `indexing` option isn't supplied. 2 is meant as a placeholder if nothing is specified and shouldn't be an option for the user.